CHANGES/19.breaking.rst

```diff
@@ -0,0 +1 @@
+Moved ``under_cached_property`` and ``cached_property`` to ``propcache.api`` -- by :user:`bdraco`.
```

Comment:
This should hopefully work:
```suggestion
Moved :func:`under_cached_property` and :func:`cached_property` to ``propcache.api`` -- by :user:`bdraco`.
```

Reply:
Hmm. that didn't seem to work

Reply:
Yeah, I realized it when I saw the RTD log. This is because these should exist as identifiers in Sphinx, and it should be able to resolve them into documented objects internally..